.github/workflows/dev-deploy-dashboard.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy DEV DASHBOARD
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+      - main
+    paths:
+      - 'apps/web/**'
+      - 'apps/dashboard/**'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test_dashboard:
+    uses: ./.github/workflows/reusable-dashboard-e2e.yml
+    with:
+      ee: true
+    secrets: inherit
+
+  deploy_dashboard:
+    needs: test_dashboard
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    uses: ./.github/workflows/reusable-dashboard-deploy.yml
+    with:
+      environment: Development
+      react_app_api_url: https://api.novu-staging.co
+      react_app_ws_url: https://dev.ws.novu.co
```

Comment:
These are not required yet, we can remove them.

Reply:
@SokratisVidros these are all needed for the web app, because we need to build it first and embed in the dashboard bundle